main.py

```diff
@@ -111,7 +111,7 @@ def registration():
         is_civilian = bool(request.form.get("is_civilian", False))
```

Comment:
I think it might be better practice to explicitly declare something as a `bool` using type annotations.
So perhaps lines like these should be written like so;
```py
is_civilian: bool = request.form.get("is_civilian", False)
```